Ryujinx.HLE/HOS/Services/Fs/FileSystemProxy/IFileSystem.cs

```diff
@@ -206,18 +123,12 @@ public ResultCode OpenDirectory(ServiceCtx context)
 
             string name = ReadUtf8String(context);
 
-            try
-            {
-                LibHac.Fs.IDirectory dir = _fileSystem.OpenDirectory(name, mode);
+            Result rc = _fileSystem.OpenDirectory(out LibHac.Fs.IDirectory dir, name, mode);
```

Comment:
```suggestion
            Result resultCode = _fileSystem.OpenDirectory(out LibHac.Fs.IDirectory dir, name, mode);
```
and add an empty line after.